server/services/reportService.ts

```diff
@@ -81,6 +86,46 @@ export default class ReportService {
       throw new Error(`Report: '${reportId}' does not exist`)
     }
 
-    await this.incidentClient.deleteReport(reportId)
+    await this.incidentClient.deleteReport(username, reportId)
+  }
+
+  private getUpdatedReport(
```

Comment:
this has been moved over from the review service.. seemed a bit weird that a service that was all about the reading reports should be doing updates